pkg/server/message.go

```diff
@@ -78,6 +78,16 @@ func (t *TranslateServiceServer) CreateMessages(
 		return nil, status.Errorf(codes.AlreadyExists, "messages already exist for language: '%s'", params.messages.Language)
 	}
 
+	// If the messages are not the original messages, translate them.
+	if !params.messages.Original {
```

Comment:
It would make it harder to use the interface, it deviates from Resource Oriented Design. There are other ways we can look at the problem. Also, we don't want to solve all the issues now.